docs/index.md

```diff
@@ -35,3 +35,5 @@
 * [Deploy ScalarDL on AWS](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/ManualDeploymentGuideScalarDLOnAWS.md)
 * [Deploy ScalarDL on Azure](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/ManualDeploymentGuideScalarDLOnAzure.md)
 
+## Schema Loader
+* [How to use ScalarDL Schema Loader](schema-loader.md)
```

Comment:
Can you move it to above the `Trouble-shooting Guide`?

Reply:
@feeblefakie 

Sure. I reorder the index in https://github.com/scalar-labs/scalardl/pull/43/commits/5036b0ba4c4b9230a4251b271b8a3d5b0998d258

Please take a look 🙇